NachoSpigot-API/src/main/java/org/bukkit/UnsafeValues.java

```diff
@@ -30,4 +30,13 @@
     Achievement getAchievementFromInternalName(String name);
 
     List<String> tabCompleteInternalStatisticOrAchievementName(String token, List<String> completions);
+
+    // Paper start
```

Comment:
No longer need  to add // paper start as we have no need to update

Reply:
It should still be kept to keep track of modified files.

Reply:
CC is easier to read and write, eg
ChatColor.BOLD.toString() + ChatColor.BLUE + "My Message" vs CC.bBlue

Reply:
That's what git is used for

Reply:
Either way, it's faster than looking through the history of commits for a file, especially while in an IDE.